docs/pages/inboxes/send-messages.md

```diff
@@ -420,6 +484,22 @@ For example:
 
 :::code-group
 
+```tsx [Browser]
+// Update disappearing message settings
+await conversation.updateMessageDisappearingSettings(1738620126404999936n, 1800000000000000n)
+
+// Clear disappearing message settings
+await conversation.removeMessageDisappearingSettings()
+```
+
+```tsx [Node]
+// Update disappearing message settings
+await conversation.updateMessageDisappearingSettings(1738620126404999936n, 1800000000000000n)
```

Comment:
sorry for my earlier feedback, this is just a number in the Node SDK, so you can remove the `n` at the end.